docs/framework/windows-workflow-foundation/wfc-exe-workflow-compiler-tool.md

```diff
@@ -0,0 +1,117 @@
+ms.date: "10/10/2020"
+helpviewer_keywords: 
+  - "wfc [Workflow]"
+  - "compiler tool"
+  - "wfc.exe"
+  - "Workflow, compilation"
+  - "Workflow, XOML files"
+  - "Workflow, wcf"
+---
+# wfc.exe (Workflow Command-line Compiler Tool)
+> [!NOTE]
+>  [!INCLUDE[DeprecatedContent](~/includes/deprecatedcontent-md.md)]  
+
+The wfc.exe workflow command-line compiler tool works with old workflow markup files that have the file extension *.xoml* (obsoleted).
+
+## Compilation process
+
+When workflows are compiled, the following procedures are performed as part of the compilation process:
+
+- Validation is performed to ensure that the workflow activities validate based on the rules that the activities have set for themselves. If there are validation errors, the compiler returns a list of the errors.  
+      
+  - A partial class is generated from the markup definition that is input into the compiler.  
+      
+  - Code is generated to help with the run-time execution of the activities. Event subscriptions are generated, which help activities know when the activities they contain are finished executing.  
+      
+  - The partial classes generated from the markup file and the partial classes from the code file are entered into the .NET Framework C\# or Visual Basic compiler. The output of this process is the .NET assembly, WorkflowSample.dll. This can be deployed to run the workflow.  
+      
+
+### Compiler Options
+
+This section shows the options for the wfc.exe workflow command-line compiler.
+
+    Microsoft (R) Windows Workflow Compiler version 3.0.0.0
+    Copyright (C) Microsoft Corporation 2005. All rights reserved.
+    
+                      Windows Workflow Compiler Options
+    
+    wfc.exe <Xoml file list> /target:assembly [<vb/cs file list>] [/language:...] 
+            [/out:...] [/reference:...] [/library:...] [/debug...] [/nocode...] 
+             [/checktypes...] [/resource:<resource info>]
+    
+                            - OUTPUT FILE -
+    /out:<file>             Output file name
+    /target:assembly        Build a Windows Workflow assembly (default).
+                            Short form: /t:assembly
+    /target:exe             Build a Windows Workflow application.
+                            Short form: /t:exe
+    /delaysign[+|-]         Delay-sign the assembly using only the public portion
+                            of the strong name key.
+    /keyfile:<file>         Specifies a strong name key file.
+    /keycontainer:<string>  Specifies a strong name key container.
+    
+                            - INPUT FILES -
+    <Xoml file list>        Xoml source file name(s).
+    <vb/cs file list>       Code-beside file name(s).
+    /reference:<file list>  Reference metadata from the specified assembly file(s).
+                            Short form is `/r:`.
```

Comment:
```suggestion
                            Short form is '/r:'.
```

Since this is just output from the compiler, I'm reverting my suggestion here. But, can you put the whole block in triple backticks?

```output

```